test/Java8andUp/src/org/openj9/test/annotationClassLoader/Test_ClassLoader.java

```
@@ -70,4 +76,83 @@ protected Class loadClass(String className, boolean resolve) throws ClassNotFoun
+				 *  [18] java.lang.reflect.Method.invoke [Bootstrap ClassLoader]
+				 *  [19] java.io.ObjectStreamClass.invokeReadObject [Bootstrap ClassLoader]
+				 *  [20] java.io.ObjectInputStream.readSerialData [Bootstrap ClassLoader]
+				 *  [21] java.io.ObjectInputStream.readOrdinaryObject [Bootstrap ClassLoader]
+				 *  [22] java.io.ObjectInputStream.readObject0 [Bootstrap ClassLoader]
+				 *  [23] java.io.ObjectInputStream.readObjectImpl [Bootstrap ClassLoader]
+				 *  [24] java.io.ObjectInputStream.readObject [Bootstrap ClassLoader]
+				 *  [25] com.sun.rowset.CachedRowSetImpl.createCopy [Platform ClassLoader]
+				 *  [26] .... [Application ClassLoader]
+				 *  
+				 * ObjectInputStream.resolveClass makes a call to JVM_LatestUserDefinedLoader.
+				 * ClassLoader returned from JVM_LatestUserDefinedLoader is passed to Class.forName.
+				 *  
+				 * CustomSyncProvider is loaded by the Application ClassLoader. CachedRowSetImpl.createCopy
+				 * will try to search for CustomSyncProvider using the ClassLoader returned by
+				 * JVM_LatestUserDefinedLoader. If JVM_LatestUserDefinedLoader returns a Bootstrap
+				 * or Platform ClassLoader, then CustomSyncProvider won't be found.
+				 * CachedRowSetImpl.createCopy will catch the ClassNotFoundException and throw
+				 * a SQLException.
+				 *  
+				 * The purpose of this test is to verify that JVM_LatestUserDefinedLoader returns
+				 * a valid Application ClassLoader. If a Bootstrap or a Platform ClassLoader is
+				 * returned, then SQLException will be thrown by CachedRowSetImpl.createCopy
+				 * since CustomSyncProvider won't be found.
+				 *  
+				 * CachedRowSetImpl.createCopy may also throw SQLException for other reasons.
+				 * Use a debugger to confirm if SQLException is a result of ClassNotFoundException
+				 * or something else.
+				 */
+				CachedRowSet crsCopy = crs.createCopy();
+			} catch (Exception e) {
```

Again, let this throw.

---

fixed.